Meta/CMake/code_generators.cmake

```diff
@@ -2,7 +2,7 @@
 # Functions for generating sources using host tools
 #
 
-function(compile_gml source output string_name)
+function(stringify_gml source output string_name)
```

Comment:
Let's do this upfront please so we don't have to introduce "new_compile_gml" temporarily